src/interlocked_hl.c

```diff
@@ -163,6 +163,53 @@ IMPLEMENT_MOCKABLE_FUNCTION(, INTERLOCKED_HL_RESULT, InterlockedHL_WaitForNotVal
     return result;
 }
 
+IMPLEMENT_MOCKABLE_FUNCTION(, INTERLOCKED_HL_RESULT, InterlockedHL_CompareExchangeIf, int32_t volatile_atomic*, target, int32_t, exchange, INTERLOCKED_COMPARE_EXCHANGE_IF, compare, int32_t*, original_target)
```

Comment:
>InterlockedHL_CompareExchangeIf [](http://example.com/codeflow?start=53&length=31)

possibly missing "reals" for this new function.

Reply:
reals added.